openduck-py/openduck_py/response_agent.py

```diff
@@ -486,8 +485,20 @@ async def speak_response(
                 meta={"text": normalized},
                 latency=t_normalize - t_chat,
             )
-
             audio_bytes_iter = _inference(normalized)
+        elif self.tts_config.provider == "gptsovits":
+            normalized = await _normalize_text(response_text)
```

Comment:
Does GPT sovits require text normalization or does the GPT part handle it implicitly?

normalization is very slow so it would be cool to skip this if it's not needed.

Reply:
ooh fair point! quick test suggests that yes it does normalize yay